### PR TITLE
Update create-a-phone-system-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updating to provide clarification on Longest Idle and Presence Based Routing

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -54,17 +54,35 @@ This video shows a basic example of how to create a call queue in Teams.
...
-### Greetings and music on hold in queue
+### 3. Assign calling ID
+
+![Screenshot of calling ID settings](media/call-queue-callingID.png)
+
+Click **Add**, search for the resource accounts that you want to allow agents to for calling ID purposes when making outbound calls, click **Add**, and then click **Add**.
+
+> [!NOTE]
+> Only resource accounts with phone numbers assigned may be used here.
+> <br>This feature is currently only available to agents in call queues that use a Teams channel. To set the outbound calling ID for agents in non-Teams channel call queues please see [Caller ID for outbound calls](create-a-phone-system-call-queue.md#caller-id-for-outbound-calls)
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:75]:
```suggestion
> <br>This feature is currently only available to agents in call queues that use a Teams channel. To set the outbound calling ID for agents in non-Teams channel call queues please see [Caller ID for outbound calls](create-a-phone-system-call-queue.md#caller-id-for-outbound-calls).
```

File: Teams/create-a-phone-system-call-queue.md [hunk 2]
```diff
@@ -130,11 +148,14 @@ Agents' Teams accounts need to be set to Teams-only mode. Agents who don't meet
 
 ![Screenshot of routing, opt out, and alert time settings](media/call-queue-presence-agents-time.png)
 
-
 **Presence-based routing** uses the availability status of call agents to determine whether an agent should be included in the call routing list for the selected routing method. Call agents whose availability status is set to **Available** are included in the call routing list and can receive calls. Agents whose availability status is set to any other status are excluded from the call routing list and won't receive calls until their availability status changes back to **Available**. 
 
 You can enable presence-based call routing with any of the routing methods.
 
+> [!NOTE]
+> When Longest idle is selected as the routing method, presence-based routing is required and automatically enabled even though the Presence-based routing toggle will be Off.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:156]:
```suggestion
> When **Longest idle** is selected as the routing method, presence-based routing is required and automatically enabled even though the Presence-based routing toggle will be **Off**.
```

File: Teams/create-a-phone-system-call-queue.md [hunk 3]
```diff
@@ -130,11 +148,14 @@ Agents' Teams accounts need to be set to Teams-only mode. Agents who don't meet
...
+> The Presence-based routing toggle will not be changeable when Longest idle is the selected routing method.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:157]:
```suggestion
> The Presence-based routing toggle will not be changeable when **Longest idle** is the selected routing method.
```

File: Teams/create-a-phone-system-call-queue.md [hunk 4]
```diff
@@ -175,7 +196,7 @@ When you have selected your call timeout options, click **Save**.
 
 ## Caller ID for outbound calls
 
-Since agents in a call queue may dial out to return a customer call, consider setting the caller ID for members of a call queue to the service number of an appropriate auto attendant. See [Manage caller ID policies in Microsoft Teams](caller-id-policies.md) for more information.
+If your queues are not using a Teams channel to control agent membership consider setting the caller ID for members of a call queue to the service number of the call queue or appropriate auto attendant. See [Manage caller ID policies in Microsoft Teams](caller-id-policies.md) for more information.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:199]:
```suggestion
If your call queues are not using a Teams channel to control agent membership, consider setting the caller ID for members of a call queue to the service number of the call queue or appropriate auto attendant. See [Manage caller ID policies in Microsoft Teams](caller-id-policies.md) for more information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@CLYVR - a few minor suggestions. I'll do a final copy edit when the status has been changed from "Draft".

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
These updates are being handled through a different PR.

**Pull request closed without merging**